src/solver/infeasible-problem-analysis/report.cpp

```diff
@@ -114,14 +116,12 @@ void InfeasibleProblemReport::logSuspiciousConstraints()
     {
         Antares::logs.error() << "* Binding constraints,";
     }
-
-    Antares::logs.error() << "* Negative hurdle costs on lines with infinite capacity (rare).";
 }
 
 void InfeasibleProblemReport::prettyPrint()
 {
-    sortConstraintsByType();
```

Comment:
The call to `sortConstraintsByType()` was moved above, with another call to a sort function.
 Besides, this call is not related to calling function's name, which is supposed to print only.